indico/modules/events/registration/fields/__init__.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core import signals
+from indico.util.signals import named_objects_from_signal
+
+
+def get_field_types():
```

Comment:
Since the signal is not really helpful (as you can't just define a new field without changing core code) I wonder if we should use it at all and not simply go for something like this:

```
def get_field_types():
    return {field.name: field for field in (FreeTextField, TextField, NumberField, TextAreaField, SelectField,
                        CheckboxField, DateField, BooleanField, PhoneField, CountryField, FileField)}
```